CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v0.13.0 (19.04.2021)
+
+Release Candidate 1
```

Comment:
Let's call it "1.0 Release Candidate 1". Might be clearer what we mean by release